test/TestApps/PushNotificationsTestApp/PushNotificationsTestApp.vcxproj

```diff
@@ -153,7 +153,7 @@
       <PrecompiledHeader>Use</PrecompiledHeader>
       <WarningLevel>Level3</WarningLevel>
       <SDLCheck>true</SDLCheck>
-      <AdditionalIncludeDirectories>%(AdditionalIncludeDirectories);$(OutDir)\..\WindowsAppRuntime_DLL;$(OutDir)\..\WindowsAppRuntime_BootstrapDLL;..\..\inc</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>$(RepoRoot)\Dev\Common;%(AdditionalIncludeDirectories);$(OutDir)\..\WindowsAppRuntime_DLL;$(OutDir)\..\WindowsAppRuntime_BootstrapDLL;..\..\inc</AdditionalIncludeDirectories>
```

Comment:
NIT: `dev` (not `Dev`)

Ultra-NITy. Is my inner OCD showing? :P

Reply:
Ok. Will fix to lower cased